pkg/manifests/rte/rte.go

```diff
@@ -185,14 +185,13 @@ func GetManifests(plat platform.Platform, version platform.Version, namespace st
 	var err error
 	mf := New(plat)
 
-	if plat == platform.OpenShift || plat == platform.HyperShift {
-		if withCustomSELinuxPolicy {
-			mf.MachineConfig, err = manifests.MachineConfig(manifests.ComponentResourceTopologyExporter, version, withCRIHooks)
-			if err != nil {
-				return mf, err
-			}
+	if plat == platform.OpenShift && withCustomSELinuxPolicy {
```

Comment:
do we prefer t he condition to be `plat == platform.OpenShift` or `plat != platform.Hypershift` ?
I'm asking because it was mentioned earlier that platform.Kubernetes is allows with MCP installed so I wonder if we want to still consider this case

Reply:
Good point. The best option is to review all the platform checks and make sure we are consistent. This change seems to follow the spirit of the old code so should be ok.

Reply:
This MachineConfig intended to install the custom SELinux policy which tied up to the RHEL OS which is running on the system. 
This is OCP specific